src/containers/ReportsTable.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {Table} from 'react-bootstrap';
+
+const ReportsTable = (props)=>{
+    const Months = ['Jan.', 'Feb.', 'Mar.', 'Apr.', 'May', 'June', 'July', 'Aug.', 'Sept.', 'Oct.', 'Nov.', 'Dec.'];
```

Comment:
The month constant seems to repeat. What do you think , may be it's better to put it to parent element not to double code?